rebootmgr/main.py

```diff
@@ -313,6 +313,23 @@ def ensure_configuration(con, hostname, dryrun) -> bool:
     return False
 
 
+def do_set_global_stop_flag(con, dc):
+    reason = "Set by " + os.environ.get("LOGNAME", "(unknown)")\
+             + " " + str(datetime.datetime.now())
+    con.kv.put("service/rebootmgr/stop", reason, dc=dc)
+    LOG.warning("Set %s global stop flag: %s", dc, reason)
+
+
+def do_set_local_stop_flag(con, hostname):
+    reason = "Node disabled by " + os.environ.get("LOGNAME", "(unknown)")\
```

Comment:
`reason = "Node disabled by " + os.getlogin()\`
in case you are root it will not show the original user.